docs/design.md

```diff
@@ -181,12 +189,15 @@ never be decreased. The core difference between the two isolation levels
 SI and SSI is that the former allows the transaction's candidate
 timestamp to increase and the latter does not.
 
-Timestamps are a combination of both a physical and a logical component
+Each cockroach node maintains a hybrid logical clock (HLC) as discussed
+in the [*Hybrid Logical Clock
+paper.(HLC)*](http://www.cse.buffalo.edu/tech-reports/2014-04.pdf) 
+A HLC is a combination of both a physical and a logical component
 to support monotonic increments without degenerate cases causing
-timestamps to diverge from wall clock time, following closely the
-[*Hybrid Logical Clock
-paper.*](http://www.cse.buffalo.edu/tech-reports/2014-04.pdf)
-
+HLC time to diverge dramatically from wall clock time. Cockroach
+picks a Timestamp for a transaction using HLC time. The timestamp
+at a node referred in this design is the HLC time at that node. 
```

Comment:
Add that throughout the document, "timestamp" always refers to the timestamp of a hybrid logical clock which is a singleton on each node and updated on a per-request basis. Whenever we mean the local walltime instead (very rarely the case), it needs to be qualified as such.